hack/build/bazel-build-builder.sh

```diff
@@ -47,7 +41,7 @@ set -e
 # to use DOCKER_PREFIX as it is set in config.sh and used elsewhere in
 # the build system, to introduce a little more consistency
 
-if [ $diffret -ne 0 ] || [ x"${ADHOC_BUILDER}" != "x" ]; then
+if [ x"${ADHOC_BUILDER}" != "x" ]; then
```

Comment:
I think we can also drop `ADHOC_BUILDER`, no? it was only added since the git diff condition was blocking the publish

Reply:
makes sense to me
I also don't find any other reference to this

Reply:
https://github.com/kubevirt/containerized-data-importer/pull/3251